plugin/v3/plugin.proto

```diff
@@ -102,6 +109,38 @@ message Read {
   }
 }
 
+message TableRelation {
+  string table_name = 1;
+  string parent_table = 2;
+}
+
+
+
+message Predicate {
+  enum Operator {
+    EQ = 0;
+    // LT = 1;
+    // LTE = 2;
+    // GT = 3;  
+    // GTE = 4;
+  }
+
+  Operator operator = 1;
+  string column = 2;
+  // marshalled arrow.Record
+  bytes record = 3;
+}
+
+message PredicatesGroup {
+  enum GroupingType {
+    AND = 0;
```

Comment:
we can start with only `AND` as we're treating this as `remove row by PK/full row contents`

Reply:
So what is the path forward to support OR grouping?